apps/ledger-live-desktop/src/renderer/modals/EditTransaction/Body.tsx

```diff
@@ -0,0 +1,290 @@
+import React, { useCallback, useState } from "react";
+import { BigNumber } from "bignumber.js";
+import { connect } from "react-redux";
+import { compose } from "redux";
+import { createStructuredSelector } from "reselect";
+import { Trans, withTranslation, TFunction } from "react-i18next";
+import { UserRefusedOnDevice } from "@ledgerhq/errors";
+import { addPendingOperation, getMainAccount } from "@ledgerhq/live-common/account/index";
+import { getAccountCurrency } from "@ledgerhq/live-common/account/helpers";
+import useBridgeTransaction from "@ledgerhq/live-common/bridge/useBridgeTransaction";
+import { Account, AccountLike, Operation, TransactionCommonRaw } from "@ledgerhq/types-live";
+import { Transaction } from "@ledgerhq/live-common/generated/types";
+// eslint-disable-next-line no-restricted-imports
+import { EIP1559ShouldBeUsed } from "@ledgerhq/live-common/families/ethereum/transaction";
+// eslint-disable-next-line no-restricted-imports
+import { TransactionRaw as EthereumTransactionRaw } from "@ledgerhq/live-common/families/ethereum/types";
+import { isEditableOperation } from "@ledgerhq/live-common/operation";
+import logger from "~/renderer/logger";
+import Stepper from "~/renderer/components/Stepper";
+import { SyncSkipUnderPriority } from "@ledgerhq/live-common/bridge/react/index";
+import { closeModal, openModal } from "~/renderer/actions/modals";
+import { accountsSelector } from "~/renderer/reducers/accounts";
+import { updateAccountWithUpdater } from "~/renderer/actions/accounts";
+import { getCurrentDevice } from "~/renderer/reducers/devices";
+import { Device } from "@ledgerhq/live-common/hw/actions/types";
+import StepMethod, { StepMethodFooter } from "./steps/StepMethod";
+import StepFees, { StepFeesFooter } from "./steps/StepFees";
+import StepConnectDevice from "~/renderer/modals/Send/steps/StepConnectDevice";
+import StepSummary, { StepSummaryFooter } from "~/renderer/modals/Send/steps/StepSummary";
+import { fromTransactionRaw } from "@ledgerhq/live-common/transaction/index";
+import StepConfirmation, {
+  StepConfirmationFooter,
+} from "~/renderer/modals/Send/steps/StepConfirmation";
+import { St, StepId } from "./types";
+
+export type Data = {
+  account: AccountLike | undefined | null;
+  parentAccount: Account | undefined | null;
+  recipient?: string;
+  amount?: BigNumber;
+  disableBacks?: string[];
+  transaction?: Transaction;
+  onConfirmationHandler: Function;
+  onFailHandler: Function;
+  transactionRaw: TransactionCommonRaw;
+  transactionSequenceNumber: number;
```

Comment:
Why is this as a separate field ? Shouldn't we have this from the `transactionRaw` ?

Reply:
fixed. Please review again. Thanks